docs/pages/components/SearchField.mdx

```diff
@@ -55,6 +55,16 @@ By default, SearchField is wrapped in a `<form role="search">` to provide better
 </form>
 ```
 
+### SearchField with trailingChildren
```

Comment:
Nit:

```suggestion
### SearchField with Trailing Children
```

Also, can we describe a little more about the use cases here about when/why we would want to use this?